notices.json

```diff
@@ -1,16 +1,16 @@
 [
   {
-    "id": "desktop_upgrade_v5.0",
+    "id": "desktop_upgrade_v5.1",
     "conditions": {
       "audience": "all",
       "clientType": "desktop",
-      "desktopVersion": ["<5.0"],
+      "desktopVersion": ["<5.1"],
       "serverVersion": [">=6.0"]
     },
     "localizedMessages": {
       "en": {
         "title": "New desktop version available",
-        "description": "Upgrade now to access a redesigned title bar to seamlessly work in Channels, Playbooks, and Boards across multiple servers with minimal context switching.",
+        "description": "Upgrade now to access desktop application automatic updates.",
```

Comment:
@cwarnermm any thoughts on better copy here that will get people excited about downloading the new desktop app with autoupdates for any future updates? ie it will be the last time they have to update manually.

Reply:
Thanks, @esethna. How about something along these lines?

title: The Desktop App now updates itself automatically!
description: Update your Desktop App one last time to start getting new features as they become available.